database-mysql/index.js

```diff
@@ -1 +1,33 @@
 // this is where the database connection will happen 
+const Sequelize = require('sequelize');
+
+const host = 'localhost';
+const user = 'root';
+const port = 3306;
+const password = '';
+const database = 'joke-me-off';
+
+const connection = new Sequelize(database, user, password, {
+  dialect: 'mysql',
+  host,
+  port,
+});
+
+const Users = connection.define('users', {
+    name: Sequelize.STRING,
+    user_name: Sequelize.STRING,
+    profile_pic: Sequelize.STRING.BINARY
+})
+const Messages = connection.define('messages', {
+    text: Sequelize.STRING,
```

Comment:
we talked about changing text to 'content' and adding a contentType property as well, so our front end knows how to render each message